Makefile

```diff
@@ -0,0 +1,8 @@
+build:
+	docker build -t europe-west2-docker.pkg.dev/ssdc-rm-ci/docker/ssdc-rm-ui-tests .
+
+install:
+	npm install cypress@9.6.0
```

Comment:
I think this should just be an `npm install`, the required package should be specified in the package.json not in this command.